pallas-txbuilder/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+use pallas_traverse::wellknown::GenesisValues;
+
+mod builder;
+mod fee;
+mod transaction;
+
+pub mod prelude;
+
+#[derive(Debug, Clone)]
+pub struct NetworkParams {
+    pub genesis_values: GenesisValues,
+    pub min_utxo_value: u64,
```

Comment:
Heads up I believe `min_utxo_value` is no longer involved in calculating tx fees (and no longer present in the protocol params at all). It's been replaced by [utxoCostPerByte](https://github.com/input-output-hk/cardano-node/blob/5006b56fc73a43a5363b5a380af0c98b86af4589/bench/tx-generator/data/protocol-parameters-v8.json#L384) and [the calculation is now based on the size of the CBOR.](https://cips.cardano.org/cips/cip55/#thenewminimumlovelacecalculation)

Reply:
Oh, I'm not using it for fee calculation, but to validate output creation.